docker/README.md

```diff
@@ -0,0 +1,20 @@
+# Docker
+
+This Dockerfile builds an image for a container that runs `ganache-cli` with the necessary contracts pre-deployed. To
+build it, do:
+
+```
+docker build -t gnosis-cond-ganache .
+```
+
+And then run it:
+
+```
+docker run -it --rm gnosis-cond-ganache
+```
+
+The addresses of the contracts are saved in a file inside the container. You can see them with:
+
+```
+docker run -it --rm cat contracts_addresses.txt
```

Comment:
Q: I think it is necessary to add the name of the container? please correct me if I am wrong, like this `docker run -it --rm gnosis-cond-ganache cat contracts_addresses.txt`

Reply:
Oops, that's right. Fixed.